src/observable/ForkJoinObservable.ts

```diff
@@ -40,8 +40,98 @@ export class ForkJoinObservable<T> extends Observable<T> {
   /* tslint:enable:max-line-length */
 
   /**
-   * @param sources
-   * @return {any}
+   * Joins last values emitted by passed Observables.
+   *
+   * <span class="informal">Wait for Observables to complete and then combine last values they emitted.</span>
+   *
+   * <img src="./img/forkJoin.png" width="100%">
```

Comment:
forkJoin needs an `asDiagram` test case in its spec file, or you should remove this line. Otherwise, it would render as a broken image element on the page.